content/user/localization/orders_status.md

```diff
@@ -18,6 +18,12 @@ Zen Cart has the following default order statuses:
 New order statuses can be created in 
 [Admin > Localization > Orders Status](/user/admin_pages/localization/orders_status/). 
 
+### What is the initial status of an order?
+
+When an order is first placed, by default it is put into status "Pending."  This can be overridden by most payment modules using the dropdown labelled "Set Order Status" in the payment module's own configuration.
+
+"Pending" is a good status for [offline payment modules](/user/payment/offline/), since you haven't received the money yet at the time the order is placed. 
```

Comment:
Probably good to mention that "Pending" is the ideal status for unconfirmed/uncompleted/authorized-only payments ... before mentioning anything about "offline" modules. Even though the classic "offline" module is "money order / check / bank transfer".

Reply:
Edit: I see you've detailed more about that on other pages.

Reply:
I still like the suggestion, and will do it on my next checkin.